lib/licenses.nix

```diff
@@ -39,6 +39,11 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     url = http://developer.amd.com/amd-license-agreement/;
   };
 
+  apache2 = spdx {
+    spdxId = "AL-2.0";
```

Comment:
According to https://spdx.org/licenses/, that license does not exist.

Reply:
Didn't know spdxId was an official register so I took a random one. Apache 2 license is registered though https://spdx.org/licenses/Apache-2.0.html. I will fix it